src/CornerstoneViewport/CornerstoneViewport.js

```diff
@@ -264,6 +271,11 @@ class CornerstoneViewport extends Component {
     if (Object.keys(updatedState).length > 0) {
       this.setState(updatedState);
     }
+
+    // -- Overlay toggle
+    const { isOverlayVisible } = this.props;
```

Comment:
Let's add this up where we destructure `frameRate` and `isPlaying` from props.

Reply:
done